### DISCUSSION TITLE
Is it possible to convert BVE to JSON?

### DISCUSSION THREAD

**Discussion by clocky-dev** [stephenberry/glaze#739] [Ideas]
BVE is very good, but maybe for debugging and printing the structs json is better with prettier. 

Can you maybe create a function to convert BVE formats to json?

Best regards

**Comment by stephenberry** [comment]
Yes, this would be a use function, to convert BEVE directly into JSON. I will convert this into an issue to add support for.

**Comment by stephenberry** [comment]
Support for `glz::beve_to_json` has been added in `glaze/binary/beve_to_json.hpp`.
